src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4396,6 +4568,63 @@ export class StudyViewPageStore
         this.changeChartVisibility(uniqueKey, false);
     }
 
+    public addColorToCategories(
+        counts: ClinicalDataCount[],
+        attributeId: string,
+        getDisplayedValue?: (value: string) => string,
+        getDisplayedColor?: (value: string) => string
+    ): ClinicalDataCountSummary[] {
+        return getClinicalDataCountWithColorByClinicalDataCount(counts).map(
```

Comment:
in the name of reducing the bloat of this file we should move this to another file.  if you would rather just put a TODO we can follow up with that when do the refactor discussed a few weeks ago.  i've actually already started this.

Reply:
This function is not part of this PR although I did try to move it to util file but decided not to. The thing is it's using lots of internal data such as this.chartToUsedColors in StudyViewPageStore. I prefer to delaying it to another PR that address this.

I agree that we should have unit tests once we move it to util file.